packages/grafana-toolkit/src/config/webpack.plugin.config.ts

```diff
@@ -224,3 +228,15 @@ export const getWebpackConfig: WebpackConfigurationGetter = options => {
     optimization,
   };
 };
+
+export const loadWebpackConfig: WebpackConfigurationGetter = options => {
+  const baseConfig = getBaseWebpackConfig(options);
+
+  const customWebpackPath = path.resolve(process.cwd(), 'webpack.plugin.config.ts');
+  if (fs.existsSync(customWebpackPath)) {
+    const customConfig = require(customWebpackPath) as { getWebpackConfig: CustomWebpackConfigurationGetter };
+    return customConfig.getWebpackConfig(baseConfig, options);
```

Comment:
Having a meaningful error message in the console would be wise if the exported *thing* is not a function.

Reply:
Added error if export is not a function.